src/core/server/saved_objects/export/get_sorted_objects_for_export.ts

```diff
@@ -139,12 +141,15 @@ export async function getSortedObjectsForExport({
   });
   let exportedObjects = [...rootObjects];
```

Comment:
nit: the code would be clearer if this was `let exportedObjects = [];` since we always override these values.